iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -148,17 +171,18 @@ static LogicalResult allocateBuffersForResults(
 }
 
 // Non-conversion equivalent of the core MLIR Linalg bufferization patterns.
-static void finalizeBufferAllocation(OpBuilder &b, linalg::LinalgOp op,
-                                     ValueRange inputs, ValueRange outputs,
-                                     BlockAndValueMapping &bvm) {
+static LogicalResult finalizeBufferAllocation(OpBuilder &b, linalg::LinalgOp op,
```

Comment:
When will it fail? It seems that we always return success().

Reply:
Right now it does. But I am just making everything return `LogicalResult` so that we can propogate errors everywhere properly.

Reply:
Sorry, stale refactoring that wasnt needed. Moving it back.